presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -629,21 +629,12 @@ protected Node visitShowFunctions(ShowFunctions node, Void context)
                     .put("language", "Language")
                     .build();
 
-            Optional<Expression> predicate = Optional.empty();
-            Optional<String> likePattern = node.getLikePattern();
-            if (likePattern.isPresent()) {
-                predicate = Optional.of(new LikePredicate(
-                        identifier("function_name"),
-                        new StringLiteral(likePattern.get()),
-                        node.getEscape().map(StringLiteral::new)));
-            }
-
             return simpleQuery(
                     selectAll(columns.entrySet().stream()
                             .map(entry -> aliasedName(entry.getKey(), entry.getValue()))
                             .collect(toImmutableList())),
                     aliased(new Values(rows.build()), "functions", ImmutableList.copyOf(columns.keySet())),
-                    predicate,
+                    node.getLikePattern().map(pattern -> new LikePredicate(identifier("function_name"), new StringLiteral(pattern), node.getEscape().map(StringLiteral::new))),
```

Comment:
I would break this into multiple lines for readability.
```
node.getLikePattern().map(pattern -> new LikePredicate(
    identifier("function_name"),
    new StringLiteral(pattern),
    node.getEscape().map(StringLiteral::new))),
```